querier.go

```diff
@@ -37,6 +37,9 @@ type Querier interface {
 	// under the constraint of another label.
 	LabelValuesFor(string, labels.Label) ([]string, error)
 
+	// LabelNames returns all the unique label names present in the block in sorted order.
```

Comment:
Not yours, but can you fix new lines in this interface? Couple lines before this.